app/db.py

```
@@ -0,0 +1,61 @@
+
+class Recording(Base):
+    """Database model for audio recordings."""
+    __tablename__ = "recordings"
+
+    id = Column(Integer, primary_key=True, index=True)
+    unique_id = Column(String, unique=False, nullable=False)
+    start_time = Column(DateTime, nullable=False)
+    end_time = Column(DateTime, nullable=False)
+    transcript = Column(String, nullable=True)
+
+    class Summary(Base):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)